html/elements/input/datetime-local.json

```
@@ -18,8 +18,7 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": false,
-                "notes": "See <a href='https://bugzil.la/888320'>bug 888320</a> and <a href='https://wiki.mozilla.org/TPE_DOM/Date_time_input_types'>TPE DOM/Date time input types</a>."
+                "version_added": "93"
               },
               "firefox_android": {
                 "version_added": true
```

This appears to have shipped on Android as well

```suggestion
                "version_added": "93"
```

---

This exists in Firefox Android already. Potentially better to make a separate issue and find the actual version it was introduced? Its existed since at least 91 though i suspect potentially much earlier.

---

Just installed and tested Firefox 55 for Android and it supports this input. Can't test much older than that because it crashes on launch on my phone. But it's possible we can find the version it was added in release notes somewhere?

---

I've also just noticed it is already set to true so potentially best to just leave it at that because getting exact versions doesn't materially help anyone as support has existed for so long.

---

Good catch, @lukewarlow! Sorry for the bad suggestion, @lafeber.